_includes/css/rcssa.css

```diff
@@ -252,6 +252,166 @@ fieldset[disabled] .btn-xl.active {
     background-color: #{{ site.data.template.color.secondary }};
 }
 
+@media(max-width:767px) {
+    .title {
+        position: relative;
+        left: 5%;
+        display: table;
+    }
+    .info {
+        width: 100%;
+        height: 65px;
+        padding-left: 10%;
+        margin-bottom: 50px;
+        display: table;
+        table-layout: fixed;
+    }
+    .u_container {
+        padding-left: 1.5vw;
+    }
+
+    h2.upon {
+        font-family: Optima;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 40px;
+        line-height: 73px;
+        text-align: center;
+        /* letter-spacing: 0.1em; */
+        color: #ffffff;
+        margin-top: -3%;
+        margin-left: 10%;
+        margin-bottom: 5%;
+        margin-right: 10%;
+    }
+    .faq_container {
+        padding-left: 5%;
+        margin-top: -5%;
+    }
+    .question {
+        margin-left: 2%;
+        padding-right: 5%;
+    }
+    .collapsible {
+        font-family: Optima;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 18px;
+        line-height: 50px;
+        letter-spacing: 0.1em;
+        color: #ffffff;
+        display: inline;
+        text-align: left;
+        background-color: transparent;
+        border: none;
+    }
+}
+
+@media(min-width:768px) {
+    .title {
```

Comment:
This "title" class seems to only affect the home page title "RCSSA"(and the css moves the title to wrong position I think). Do we really need this one?